packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/components/pool-form/constants.js

```diff
@@ -2,10 +2,23 @@ export const DEFAULT_ALGORITHM = 'roundRobin';
 
 export const DEFAULT_SESSION_PERSISTENCE_TYPE = 'sourceIP';
 
-export const APP_COOKIE_SESSION_PERSISTENCE = 'appCookie';
+const persistenceTypes = ['sourceIP', 'httpCookie', 'appCookie'];
+
+export const APP_COOKIE_SESSION_PERSISTENCE = persistenceTypes[2];
+
+export const PROTOCOL_SESSION_PERSISTENCE_TYPE_COMBINATION = {
+  http: persistenceTypes,
+  https: persistenceTypes,
+  tcp: persistenceTypes,
+  proxy: persistenceTypes,
+  proxyV2: persistenceTypes,
+  udp: [persistenceTypes[0]],
+  sctp: [persistenceTypes[0]],
+};
```

Comment:
Sorry, I didn't pay attention that some properties was already existing

```suggestion
export const DEFAULT_SESSION_PERSISTENCE_TYPE = 'sourceIP';
eexport const APP_COOKIE_SESSION_PERSISTENCE = 'appCookie';
const persistenceTypes = [
   DEFAULT_SESSION_PERSISTENCE_TYPE,
   'httpCookie', 
   APP_COOKIE_SESSION_PERSISTENCE
 ];

export const PROTOCOL_SESSION_PERSISTENCE_TYPE_COMBINATION = {
  http: persistenceTypes,
  https: persistenceTypes,
  tcp: persistenceTypes,
  proxy: persistenceTypes,
  proxyV2: persistenceTypes,
  udp: [DEFAULT_SESSION_PERSISTENCE_TYPE],
  sctp: [DEFAULT_SESSION_PERSISTENCE_TYPE],
};
```